### ISSUE TITLE
Remove Points Hooks admin screen

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpoints#389]
This is a long-term project that will take some preparation. Some details on that can be found in https://github.com/WordPoints/hooks-api/issues/23. In part what we'd like to do is get better parity between the old points hooks and the new hook events.

**Comment by JDGrimes** [comment]
See also: https://github.com/WordPoints/hooks-api/issues/31, https://github.com/WordPoints/hooks-api/issues/45

**Comment by JDGrimes** [comment]
When we do this it would be a good idea to update the old points logs meta keys to match the new format. See https://github.com/WordPoints/wordpoints/issues/435#issuecomment-237045570.

**Comment by JDGrimes** [comment]
I guess the above comment goes for the log types as well, and is really moot without that, since the log types are all changed.

**Comment by JDGrimes** [comment]
When we do this we'll want to import the non auto-reversing hooks. I've pasted a test case for non auto-reversing hooks [in this gist](https://gist.github.com/JDGrimes/442cfdbcc9710043a369e4cef5b60c26).

**Comment by JDGrimes** [comment]
Probably this should be done before features like [per-post comment points](https://trello.com/c/IRlnD5z0), because otherwise it will be annoying for users who are still using Points hooks to have to manually import in order to use these new features. And if they manually import (without us providing them a button, that is), the reactions won't behave properly (that is, they won't be coordinated with the old points hooks they are "imported" from). So I guess we should really make importing the hooks a higher priority.

**Comment by JDGrimes** [comment]
The above comment really applies to import of the affected hook types, not to removing the screen completely.

Speaking of which, I've just added a basic outline of steps and list of features that we need to achieve parity with to the main comment. I think that we need to create a ticket for each part of this, like a ticket for importing the hooks, and one for each thing that needs to be done to achieve feature parity. That will help to keep this ticket from getting unmanageable.

**Comment by JDGrimes** [comment]
#257 is another issue to consider with feature parity, I think.
